storage/pruning/pruningStorer.go

```diff
@@ -371,6 +399,35 @@ func (ps *PruningStorer) Get(key []byte) ([]byte, error) {
 	return nil, fmt.Errorf("key %s not found in %s", hex.EncodeToString(key), ps.identifier)
 }
 
+// GetFromOldEpochsWithoutCache searches the old epochs for the given key without updating the cache
+func (ps *PruningStorer) GetFromOldEpochsWithoutCache(key []byte) ([]byte, error) {
+	v, ok := ps.cacher.Get(key)
+	if ok {
+		buff, isByteSlice := v.([]byte)
+		if isByteSlice {
+			return buff, nil
+		}
+	}
+
+	ps.lock.RLock()
+	defer ps.lock.RUnlock()
+
+	if ps.bloomFilter != nil && !ps.bloomFilter.MayContain(key) {
```

Comment:
we should have a cleanup task for these bloom filters as we do not use them and continue to clutter the new code.

Reply:
Added a todo to remove the bloom filter.